CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to the project
+
+
+<summary>Instructions for development</summary>
```

Comment:
As is no longer nested we can remove `<summary>` tags.

Reply:
Edited.